src/services/services.ts

```diff
@@ -4965,8 +4965,8 @@ namespace ts {
             // Compute the meaning from the location and the symbol it references
             let searchMeaning = getIntersectingMeaningFromDeclarations(getMeaningFromLocation(node), declarations);
 
-            // Get the text to search for, we need to normalize it as external module names will have quote
-            let declaredName = getDeclaredName(symbol, node);
+            // Get the text to search for, we need to normalize it as external module names will have quotes
+            let declaredName = getDeclaredName(typeChecker, symbol, node);

```

Comment:
What you mean by normalization? stripping quotes in ```getDeclaredName```? if so could you mention this in the comment